sonic-net/sonic-buildimage
# Fix distclean for bookworm

#### Why I did it

`make distclean` is broken because FIPS support is not yet available on Bookworm

```
make distclean
+++ --- Making distclean --- +++
./scripts/run_with_retry make BLDENV=bookworm -f Makefile.work distclean
make[1]: Entering directory '/home/konstantin/Develop/sonic-buildimage'
Makefile.work:195: FIPS support not yet available on Bookworm
...
make: *** No rule to make target 'target/debs/bookworm/symcrypt-openssl_0.10_amd64.deb-clean', needed by 'target/debs/bookworm/openssl_1.1.1n-0+deb11u5+fips_amd64.deb-clean'.  Stop.
make[1]: *** [Makefile.work:629: distclean] Error 2
```

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Add FIPS packages to the list of `SONIC_EXTRA_DEBS` only if `INCLUDE_FIPS=y`

#### How to verify it

```
make target/docker-base-bookworm.gz
make distclean
```

Make sure distclean is finished without errors.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
